src/pyclaw/solution.py

```diff
@@ -136,8 +136,7 @@ def get_clawpack_dot_xxx(modname): return modname.rpartition('.')[0]
         if len(arg) == 1:
             # Load frame
             frame = arg[0]
-            if not isinstance(frame,int):
-                raise Exception('Invalid pyclaw.Solution object initialization')
+            frame = int(frame)
```

Comment:
Is there a reason not to just do `frame = int(arg[0])`?

Reply:
Just that the error traceback is slighly more clear:
```
    frame = int(frame)
            ^^^^^^^^^^
ValueError: invalid literal for int() with base 10: '_1_'
```

Reply:
Hmm, good point.  Not sure what is better I guess then.

Reply:
Well... 'frame' still appears in `frame = int(arg[0])` so I commited that change. Funny how it takes me 5 commits to change two lines.